js/src/utils/concatenateListOfWords.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * A function that joins a list of words using the sepators provided as the second and third argument.
+ *
+ * Example:
+ *
+ * concatenateListOfWords(['A', 'B', 'C']) will return 'A, B and C'
+ *
+ * @param  {Array<string>} listWords A list of words to concatenate.
+ * @param  {string} [separator=', '] The separator between words.
+ * @param  {string} [separatorLastElement='and'] The separator for the last word.
+ * @return {string} The concatenated string.
+ */
+
+export default function concatenateListOfWords(
+	listWords,
+	separator = ', ',
```

Comment:
💅 Note that separator like this should also be handled with i18n. Not all languages use a comma and space `, ` as separator. For example, in Chinese, a comma is `，`, and there is even an enumeration comma `、`.

More info:

- [Chinese punctuation](https://en.wikipedia.org/wiki/Chinese_punctuation)
- [Comma in languages other than Western European](https://en.wikipedia.org/wiki/Comma#Languages_other_than_Western_European) - Contains mention of enumeration or ideographic comma in Chinese, Japanese and Korean.

Reply:
Fixed in https://github.com/woocommerce/google-listings-and-ads/pull/1602/commits/340a524876435634fea495ca977d082f74f67ea3.

Reply:
Thanks @eason9487 . I added a comment about your change in https://github.com/woocommerce/google-listings-and-ads/pull/1602#discussion_r943764746.